src/hevm/test/test.hs

```diff
@@ -2379,6 +2383,52 @@ genWord litFreq sz = frequency
    subStore = genStorage (sz `div` 10)
    subByte = genByte (sz `div` 10)
 
+genWord2 :: Int -> Int -> Gen (Expr EWord)
```

Comment:
why do we need this?

Reply:
OOps, no, that was some artefact, removed.